### PR TITLE
Disable BLOB columns data export

### PR DESCRIPTION
**Pull request by sanyamsinghal**
In Oracle/MySQL, Blobs will be considered in schema but no data will be transfer for Blobs columns, so they will be empty.
In PostgreSQL, `bytea` and `text` will behave normally as other datatypes but Large Object`lo` datatype will not be exported.

Testing:
1. Tested standard sakila schema for the following changes with all Oracle and PostgreSQL.
2. Tested the same for ds2 schema having partitioned tables with Oracle and PostgreSQL.
3. Also verified the correctness of the corresponding new log lines for these changes.

### CODE REVIEW

File: yb_migrate/src/migration/postgres.go [hunk 1]
```diff
@@ -33,28 +33,6 @@ import (
 
 var commandNotFoundRegexp *regexp.Regexp = regexp.MustCompile(`(?i)not[ ]+found[ ]+in[ ]+\$PATH`)
 
-// TODO: check for pgdump and psql/ysqlsh - installed/set-in-path
-func CheckToolsRequiredForPostgresExport() {
```
Comment (kneeraj) [inline yb_migrate/src/migration/postgres.go:37]: Why this code was required to be removed for Blob removal capability?

**Review by kneeraj — APPROVED** [review]
LGTM

A couple of clarifications. Can you also update the PR with how you tested it and for what all source types you tested this. Thanks.

File: yb_migrate/src/utils/utils.go [hunk 2]
```diff
@@ -212,7 +212,7 @@ func CheckToolsRequiredInstalledOrNot(source *Source) {
 	case "oracle":
 		toolsRequired = []string{"ora2pg", "sqlplus"}
 	case "postgresql":
-		toolsRequired = []string{"pg_dump", "strings"}
+		toolsRequired = []string{"pg_dump", "strings", "pg_restore"}
```
[the remark this answers is not in the document]
Comment (sanyamsinghal) [reply yb_migrate/src/utils/utils.go:215]: Its not related to this PR. In general, we are using pg_restore so there should be a check for it.

File: yb_migrate/src/migration/postgres.go (hunk 1 above)
Comment (sanyamsinghal) [reply yb_migrate/src/migration/postgres.go:37]: This function is stale/not-used now. Instead we are using `CheckToolsRequiredInstalledOrNot()` everywhere.

**Pull request merged**